client/payment-details/utils/tax-descriptions.ts

```diff
@@ -0,0 +1,115 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Supported tax description codes for different countries and tax types.
+ * These codes are used to identify and localize tax descriptions in the UI.
+ *
+ * Format: [Country Code] [Tax Type]
+ * Examples:
+ * - "US VAT" for United States Value Added Tax
+ * - "JP JCT" for Japan Consumption Tax
+ * - "AU GST" for Australian Goods and Services Tax
+ */
+type TaxDescriptionKey =
+	| 'AT VAT' // Austria VAT
+	| 'BE VAT' // Belgium VAT
+	| 'BG VAT' // Bulgaria VAT
+	| 'CY VAT' // Cyprus VAT
+	| 'CZ VAT' // Czech Republic VAT
+	| 'DE VAT' // Germany VAT
+	| 'DK VAT' // Denmark VAT
+	| 'EE VAT' // Estonia VAT
+	| 'ES VAT' // Spain VAT
+	| 'FI VAT' // Finland VAT
+	| 'FR VAT' // France VAT
+	| 'GB VAT' // Great Britain VAT
+	| 'GR VAT' // Greece VAT
+	| 'HR VAT' // Croatia VAT
+	| 'HU VAT' // Hungary VAT
+	| 'IE VAT' // Ireland VAT
+	| 'IT VAT' // Italy VAT
+	| 'LT VAT' // Lithuania VAT
+	| 'LU VAT' // Luxembourg VAT
+	| 'LV VAT' // Latvia VAT
+	| 'MT VAT' // Malta VAT
+	| 'NL VAT' // Netherlands VAT
+	| 'PL VAT' // Poland VAT
+	| 'PT VAT' // Portugal VAT
+	| 'RO VAT' // Romania VAT
+	| 'SE VAT' // Sweden VAT
+	| 'SI VAT' // Slovenia VAT
+	| 'SK VAT' // Slovakia VAT
+	| 'AU GST' // Australia GST
+	| 'NZ GST' // New Zealand GST
+	| 'SG GST' // Singapore GST
+	| 'CH VAT' // Switzerland VAT
+	| 'JP JCT' // Japan Consumption Tax
+	| 'default'; // Default tax description when no specific code matches
+
+/**
+ * Tax description mapping for localization.
+ * Keys are the tax descriptions from the API response.
+ * Values are the translatable strings.
+ *
+ * Note: Country codes are intentionally left untranslated as they are standardized ISO codes.
+ */
+export const taxDescriptions: Record< TaxDescriptionKey, string > = {
```

Comment:
```suggestion
const taxDescriptions: Record< TaxDescriptionKey, string > = {
```

Since it's not imported by anything, we don't need to export it.

Reply:
Removed it here [43c9d97](https://github.com/Automattic/woocommerce-payments/pull/10707/commits/43c9d9768378ba234a0d9b1c18a298082089646c) thanks!